src/lib/components/customer/event/PaginationNav.svelte

```
@@ -0,0 +1,108 @@
+  function handlePageChange(page: number) {
+    if (onPageChange) {
+      onPageChange(page);
+    }
+  }
+</script>
+
+{#if totalPages > 1}
+  <nav class="mt-12 flex items-center justify-center gap-1.5 pb-4" aria-label="Phân trang">
+    <!-- Previous -->
+    <a
+      href={buildUrl(currentPage - 1)}
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)